terraform/groups/ecs-service/data.tf

```diff
@@ -0,0 +1,68 @@
+
+data "vault_generic_secret" "stack_secrets" {
+  path = "applications/${var.aws_profile}/${var.environment}/${local.stack_name}-stack"
+}
+
+data "vault_generic_secret" "service_secrets" {
+  path = "applications/${var.aws_profile}/${var.environment}/${local.stack_name}-stack/${local.service_name}"
+}
+
+data "aws_kms_key" "kms_key" {
+  key_id = local.kms_alias
+}
+
+data "aws_vpc" "vpc" {
+  filter {
+    name   = "tag:Name"
+    values = [local.vpc_name]
+  }
+}
+
+#Get application subnet IDs
+data "aws_subnets" "application" {
+  filter {
+    name   = "tag:Name"
+    values = [local.application_subnet_pattern]
+  }
+}
+
+data "aws_ecs_cluster" "ecs_cluster" {
+  cluster_name = "${local.name_prefix}-cluster"
+}
+data "aws_iam_role" "ecs_cluster_iam_role" {
+  name = "${local.name_prefix}-ecs-task-execution-role"
+}
+
+data "aws_lb" "service_lb" {
+  name = "${var.environment}-chs-chgovuk"
```

Comment:
I think this will link to the web url rather than the api.

Route 53 shows 
cidev.aws.chdev.org -> internal-cidev-chs-chgovuk-1154682282.eu-west-2.elb.amazonaws.com and the DNS
internal-cidev-chs-chgovuk-1154682282.eu-west-2.elb.amazonaws.com points to the load balancer cidev-chs-chgovuk

[cidev-chs-chgovuk](https://eu-west-2.console.aws.amazon.com/ec2/home?region=eu-west-2#LoadBalancer:loadBalancerArn=arn:aws:elasticloadbalancing:eu-west-2:169942020521:loadbalancer/app/cidev-chs-chgovuk/7b8dbe87ca1d977d;tab=listeners)	
internal-cidev-chs-chgovuk-1154682282.eu-west-2.elb.amazonaws.com

Reply:
Good spot @bcullerton . It should be `name = "${var.environment}-chs-apichgovuk"`